### ISSUE TITLE
Two Nukies On High Pop

### ISSUE THREAD

**Issue by Kezu02** [space-wizards/space-station-14#27489]
## Description

Two rounds on Lizard had 60~ and 80 pop where there were only two nukies that spawned.

**Comment by Plykiya** [comment]
issue should be solved by https://github.com/space-wizards/space-station-14/pull/28327

**Comment by Reese1243** [comment]
we can make it if we try!

**Comment by themias** [comment]
Yeah, this looks like it should be fixed by #28327. We can reopen if it gets replicated again.
